spec/section/joins.md

```diff
@@ -201,6 +207,11 @@ If an inner joins would have been used, the logical view would have only 3 logic
 </aside>
 </aside>
 
+<aside class="issue">
+We cannot just use the term `parent.#` as a field, since there may be multiple parents and `parent` could also be a declared field.
+Do we need this as a field at all?
```

Comment:
We don't need this. I propose to take parent.# and parent.<it> out.

Reply:
done